nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2103,6 +2104,11 @@ private static ExpressionTree stripParensAndCasts(ExpressionTree expr) {
         expr = ((TypeCastTree) expr).getExpression();
         someChange = true;
       }
+
+      // Strips Nullchk operator
+      if (expr.getKind().equals(OTHER) && expr instanceof JCTree.JCUnary) {
+        expr = ((JCTree.JCUnary) expr).getExpression();
```

Comment:
```suggestion
        expr = ((JCTree.JCUnary) expr).getExpression();
        someChange=true;
```
Need to set the `someChange` variable

Reply:
Modulo adding this patch, this PR looks good to me :)